src/bda/plone/orders/mailnotify.py

```diff
@@ -415,14 +426,13 @@ def __init__(self, event):
     def __call__(self, *args):
         return self.event.booking_attrs['eventtitle']
 
-
 def notify_booking_cancelled(event, who=None):
     """Send notification mail after booking was cancelled.
     """
     order_data = OrderData(event.context, uid=get_order_uid(event))
     templates = dict()
     templates.update(get_booking_cancelled_templates(event.context))
-    templates['booking_cancelled_title_cb'] = BookingCancelledTitleCB(event)
+    templates['booking_cancelled_title_cb'] = ItemOutOfStockTitleCB(event)
```

Comment:
@jensens I guess this one is wrong. Booking cancelled != out of stock. Probably needs to be reverted.